src/org/usfirst/frc/team2399/robot/OI.java

```diff
@@ -57,6 +65,10 @@ public static double getRightY() {
 	private static Button climbUpButt = new JoystickButton(leftJoy, 86);
 	private static Button climbDownButt = new JoystickButton(leftJoy, 59);
 	private static Button climbStopButt = new JoystickButton(leftJoy, 300);
+	private static Button driveDistanceIncrementButt = new JoystickButton(shooterJoy, 5);
+	private static Button driveDistanceDecrementButt = new JoystickButton(shooterJoy, 3);
+	private static Button driveAngleIncrementButt = new JoystickButton(shooterJoy, 6);
+	private static Button driveAngleDecrementButt = new JoystickButton(shooterJoy, 4);
```

Comment:
Why are we adjusting drive constants on the shooter joystick? Not a fail, just a question.